pkg/serverless/enhanced_metrics.go

```diff
@@ -119,8 +119,7 @@ func sendTimeoutEnhancedMetric(tags []string, metricsChan chan []metrics.MetricS
 }
 
 // getTagsForEnhancedMetrics returns the tags that should be included with enhanced metrics
-func getTagsForEnhancedMetrics() []string {
-	tags := aws.GetARNTags()
+func getTagsForEnhancedMetrics(tags []string) []string {
```

Comment:
I would consider renaming this function -- it looks like it now takes a list of tags and adds a cold start tag. Just from the function name, it's not obvious what tags I need to pass in. Maybe it should just be called `addColdStartTag`?

Reply:
✅